docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,282 @@
+OrganizationalUnitIdentifier: admin
+OrdererOUIdentifier:
+Certificate: cacerts/<root CA cert for this org>.pem
+OrganizationalUnitIdentifier: orderer' > <path to msp>/msp/config.yaml
+```
+
+Or you can manually copy the Node OU material into the `config.yaml` file for the `msp` folder:
+
+```
+ClientOUIdentifier:
+  Certificate: cacerts/<root CA cert for this org>.pem
+  OrganizationalUnitIdentifier: client
+PeerOUIdentifier:
+  Certificate: cacerts/<root CA cert for this org>.pem
+  OrganizationalUnitIdentifier: peer
+AdminOUIdentifier:
+  Certificate: cacerts/<root CA cert for this org>.pem
+  OrganizationalUnitIdentifier: admin
+OrdererOUIdentifier:
+  Certificate: cacerts/<root CA cert for this org>.pem
+  OrganizationalUnitIdentifier: orderer
+```
+
+In a production scenario, it is assumed that users will be creating only one organization. However, it is a good practice to establish a separate folder structure for this organization and then create a structure underneath this organization for your `msp` (defining the organization) and your nodes (which will have a local MSP and TLS sections).
+
+If you are creating an orderer, you obviously do not need to copy the `PeerOUIdentifier` into your `config.yaml` file (or vice versa), but for the sake of simplicity you might want to use the entire section --- extra stanzas do no harm, and they allow the same `config.yaml` to be used for multiple types of nodes and identities associated with an organization.
+
+## Register an identity
+
+While identities that will be used by admins (or other users) and identities used by nodes have different purposes, they are fundamentally all just **identities**: public/private key pairs in which the public key is known to others and the private key is used to sign, generating an output which can be verified to have come from the private key even though the private key itself is never exposed.
+
+As discussed above, an identity is first registered with a CA by a CA admin. This identity is then enrolled by the user of the identity. If you are using the Fabric CA client, this registration command looks like this (regardless of the type of identity you are enrolling, and the type of CA):
+
+```
+./fabric-ca-client register -d --id.name <ID_NAME> --id.secret <ID_SECRET> -u <CA_URL> --mspdir <CA_ADMIN> --id.type <ID_TYPE> --id.attrs $ID_ATTRIBUTE --tls.certfiles <TLSCERT> --csr.hosts $CSR_HOSTNAME
+```
+
+Where the variables are the following:
+
+* `ID_NAME`: The username of the identity. This name will be given to the user out of band, who will use it when enrolling.
+* `ID_SECRET`: The secret (similar to a password) for the identity. Must be given along with the username as part of enrollment.
+* `CA_URL`: The URL of the CA, followed by the port `7054` (unless the default port has been changed).
+* `CA_ADMIN`: The path to the location of the certificates for the admin of the CA the identity is being registered with.
```

Comment:
Should this be:
The path to the location of the certificates for the admin of the CA.